engine/src/main/scala/cromwell/jobstore/JobStoreWriterActor.scala

```diff
@@ -16,6 +16,8 @@ case class JobStoreWriterActor(jsd: JobStore, dbBatchSize: Int, override val dbF
 
   implicit val ec = context.dispatcher
 
+  log.info(s"JobStoreWriterActor configured to write to the database with batch size {} and flush rate {}.", dbBatchSize, dbFlushRate)
```

Comment:
You are correct sir.  But should the observed logging of flushes inspire anyone to knobify these, the log messages are knob-ready.